src/components/Quote/Quote.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Box,
+  Container,
+  Grid,
+  IconButton,
+  Stack,
+  useTheme,
+} from '@mui/material';
+import { type CommonProps, type Generic } from 'types/components';
+import QuoteText from './QuoteText';
+import { useState } from 'react';
+import SwipeableViews from 'react-swipeable-views';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+
+export interface QuoteProps extends CommonProps, QuoteTextProps {
+  image?: string | Generic;
+  altText?: string;
+  inverse?: boolean;
+  background?: string;
+  size: string;
+  quotes: QuoteTextProps[];
+}
+interface QuoteTextProps extends CommonProps {
+  subtitle?: string | Generic;
+}
```

Comment:
You should not extends CommonProps because i see that theme isn't used.
Also I think you should change 'subtitle' with a more generic 'text' (If I see subtitle we are aspecting to have also a title, but it's not the case here)

Reply:
done